web/controllers/stripe_connect_account_controller.ex

```diff
@@ -3,15 +3,15 @@ defmodule CodeCorps.StripeConnectAccountController do
   use JaResource
 
   alias CodeCorps.StripeConnectAccount
-  alias CodeCorps.StripeService.StripeConnectAccountService
+  alias CodeCorps.StripeService.StripeManagedConnectAccountService
```

Comment:
It was just a temporary thing so I can work on a new one while still having the old one for reference.